react/MuiCozyTheme/theme.js

```diff
@@ -286,13 +287,13 @@ const makeOverrides = theme => ({
     },
     button: {
       '&$selected, &$selected:hover': {
-        backgroundColor: 'var(--zircon)'
+        backgroundColor: theme.palette.background.selected
       },
       '&:hover': {
-        backgroundColor: theme.palette.grey[100]
+        backgroundColor: theme.palette.background.banner
```

Comment:
I find it weird to use bakground.banner for something that is not a banner? Is it really wanted? Why didn't you continue to use theme.palette.grey[100] ?

Reply:
Yes I agree, it's just that it is the background color for the banner and I do nt have a good name. I'll check with joel.

Reply:
I checked with Joel and in fact, we do not need a 3rd background color, we can use background.default (ListSubHeader, Sidebar, Banner, a light gray in the normal theme, primaryColorDark in inverted theme), and background.paper (Card etc.. white in the normal theme, primaryColor in inverted theme).